packages/scoutgame/src/projects/getUserScoutProjects.ts

```diff
@@ -0,0 +1,75 @@
+import type { ScoutProjectMemberRole } from '@charmverse/core/prisma-client';
+import { prisma } from '@charmverse/core/prisma-client';
+
+import { projectInfoSelect, projectSelect } from './projectSelect';
+
+export type UserScoutProject = {
```

Comment:
User and Scout are redundant in "UserScoutProject" . can we just call this "Project" in any of our custom types?

Reply:
I think one approach could be ScoutProjectDetailed and ScoutProjectMinimal, that way it doesn't prefix anything and you know what each one is used for kinda (detailed for viewing it on the project page while minimal is required to show in the public profile pages)